paypal/paypal-messaging-components
# fix: setup global state from script in modal setup

## Description

- Modal setup will now setup global state similar to the message to account for script attributes such as `data-pp-namespace`

## Screenshots

N/A

## Testing instructions

- Use `data-pp-attribute` to confirm that it respects the value by placing the standalone modal on `window.<namespace>`
- Confirm that a value of `paypal` does not cause adverse affects
